idl/docs/imaging.md

```diff
@@ -79,8 +85,10 @@ under foxsi_get_output_2d_image.pro.
 
 The function may be called with a default source (containing image and spectral info) by:
 
-    IDL> out_maps = foxsi_get_output_image_cube()
-    IDL> plot_map, out_map
+```
+IDL> out_maps = foxsi_get_output_image_cube()
+IDL> plot_map, out_map
```

Comment:
Can you fix this line to use the variable `out_maps`, and likely also `movie_map` instead of `plot_map`?